app/src/main/cpp/src/components/transport_manager/src/bluetooth_le/ble_control_server.cc

```diff
@@ -0,0 +1,38 @@
+#include "transport_manager/bluetooth_le/ble_control_server.h"
+#include <thread>
+#include "utils/logger.h"
+
+SDL_CREATE_LOG_VARIABLE("Ble_Control_Server")
+
+namespace{
+    const int buffer_size = 185;
```

Comment:
Please use unified bigger buffer size, maybe we need to find some good place for this constant

Reply:
done in 66d56ca414ac2d17dea823a2186a8d3f2569ec83